thunder/executors/transformer_engineex.py

```diff
@@ -458,6 +468,36 @@ def _get_te_wrapper_string():
     return TE_CTX_STR
 
 
+def te_sync_fp8_meta_bwd_meta():
+    pass
+
+
+def te_sync_fp8_meta_bwd_impl():
+    FP8GlobalStateManager.reduce_and_update_fp8_tensors(forward=False)
```

Comment:
Is this available in TE version < 1.6?

Reply:
Previously it was https://github.com/NVIDIA/TransformerEngine/blob/6a9edc38bf9b941b7d369af5103fa8fe0b121d61/transformer_engine/pytorch/fp8.py#L343

As per the talks with TE team, this is the new function and should stay stable most likely.